collector.spec

```diff
@@ -0,0 +1,59 @@
+%global goipath         github.com/os-observability/redhat-opentelemetry-collector
+
+%define project_version %(make -s -C .. project_version)
+
+Version:                %{project_version}
+ExcludeArch:            %{ix86} s390 ppc ppc64 aarch64
+
+%gometa
+
+%global common_description %{expand:
+Collector with the supported components for a Red Hat build of OpenTelemetry product}
+
+%global golicenses    LICENSE
+%global godocs        README.md
+
+%define project_name %(make -s -C .. project_name)
+
+Name:           %{project_name}
+Release:        1%{?dist}
+Summary:        Red Hat build of OpenTelemetry
+
+License:        Apache-2.0
+
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-deps-%{version}.tar.gz
+
+BuildRequires: go-toolset-1.20
+BuildRequires: git
+BuildRequires: make
+
+%description
+%{common_description}
+
+%gopkg
```

Comment:
If we're planning to bundle and only package the final binary, `%gopkg` needs to be removed.